scripts/scil_bundle_meanstd.py

```diff
@@ -28,7 +28,8 @@ def _build_arg_parser():
     p.add_argument('in_bundle',
                    help='Fiber bundle file to compute statistics on')
     p.add_argument('metrics', nargs='+',
-                   help='Nifti metric(s) to compute statistics on.')
+                   help='Nifti file to compute statistics on. Probably some '
+                        'tractometry measure(s) such as FA, MD, RD, ...')
```

Comment:
Future work maybe, the words measure, metric, value, stat are used in a confusing way.